filesystems/drs/src/test/scala/cromwell/filesystems/drs/DrsPathBuilderSpec.scala

```diff
@@ -273,18 +222,24 @@ class DemoDosPathBuilderSpec extends TestKitSuite with FlatSpecLike with Matcher
   )
 
   private def badPaths = Seq(
-    BadPath("an empty path", "", " does not have a dos scheme"),
-    BadPath("a GCS path", s"gs://$bucket/hello/world", "gs://mymadeupbucket/hello/world does not have a dos scheme"),
-    BadPath("an bucketless path", "dos://", "The specified DOS path 'dos://' does not parse as a URI.\nExpected authority at index 5: dos://"),
-    BadPath("a bucket named .", "dos://./hello/world", "The path 'dos://./hello/world' does not seem to be a valid DOS path. Please check that it starts with dos:// and that the bucket and object follow DOS naming guidelines at https://cloud.google.com/storage/docs/naming."),
-    BadPath("a non ascii bucket name", "dos://nonasciibucket£€/hello/world",
```

Comment:
If these two tests of  `"dos://./hello/world"` and `"dos://nonasciibucket£€/hello/world"` are now allowed, then they should be added as `GoodPath` expectations. "Good" expectations doesn't mean the test conforms to a (non-existent) spec. "Good" just means it doesn't throw an exception, and that we will know if someone breaks the last-known expectation.